### PR TITLE
Update Phoenix example to use 1.5 and live_view

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

File: hello_phoenix/firmware/config/target.exs [hunk 1]
```diff
@@ -1,5 +1,17 @@
 import Config
 
+config :ui, UiWeb.Endpoint,
+  url: [host: "nerves.local"],
+  http: [port: 80],
+  secret_key_base: "HEY05EB1dFVSu6KykKHuS4rQPQzSHv4F7mGVB/gnDLrIu75wE/ytBXy2TaL3A6RA",
+  live_view: [signing_salt: "AAAABjEyERMkxgDh"],
+  # check_origin: ["//*.local"],
```
Comment (jjcarstens) [inline hello_phoenix/firmware/config/target.exs:8]: Should we just set this to `check_origin: false` to simplify possibly errors

File: hello_phoenix/firmware/lib/firmware/application.ex [hunk 2]
```diff
@@ -3,29 +3,45 @@ defmodule Firmware.Application do
...
+  def children(:host) do
     [
+      # Children that only run on the host
       # Starts a worker by calling: Firmware.Worker.start_link(arg)
       # {Firmware.Worker, arg},
     ]
   end
 
   def children(_target) do
+    :os.cmd('epmd -daemon')
```
Comment (jjcarstens) [inline hello_phoenix/firmware/lib/firmware/application.ex:33]: gotta have the Erlang distribution 🤣

File: hello_phoenix/firmware/config/target.exs (hunk 1 above)
Comment (ConnorRigby) [reply hello_phoenix/firmware/config/target.exs:8]: ah i didn't know that was possible. I think yes.

File: hello_phoenix/firmware/lib/firmware/application.ex (hunk 2 above)
Comment (ConnorRigby) [reply hello_phoenix/firmware/lib/firmware/application.ex:33]: ah whoops, i meant to remove that. I was testing that it worked in the live dashboard..

**Pull request merged**